wiki/faq.md

```diff
@@ -42,54 +44,61 @@ The maximum share of new Gridcoins for this project stays the same
 percentage (subject to the number of total projects). And projects can be
 removed by a blockchain vote.
 
-Security in Gridcoin is based on Proof-of-Stake.
+Security in Gridcoin is based on Proof-of-Stake. Network consensus is not dependent on BOINC rewards working properly or even existing. Even if every BOINC project shut down tomorrow, Gridcoin's blockchain would continue making new blocks and functioning as expected (sending and receiving transactions), minus the minting of rewards for BOINC users.
 
 ---
 # Crunching (Research & Mining Equivalent)
 
 ---
 ### How do I start crunching?
 Consult these guides:
-* [Solo](/guides/boinc-install.htm "sitelink")
+* [Solo](/guides/boinc-install.htm "sitelink") (recommended method)
 * [Pool](/guides/pool.htm "sitelink")
 
 ---
-### My CPID is not in the network, What do I do?
+### I get an "e-mail mismatch" or "split CPID" error in the solo crunching wizard in the wallet?
+First, make sure you are using the same e-mail address for each BOINC project you are attached to.
+
+Some context: CPIDs are IDs assigned to you by BOINC projects based on your e-mail address and publish user stats by CPID (which Gridcoin uses to calculate your rewards). When you first attach to a project, it gives you a CPID (so all projects will have different ones). Your BOINC client then picks which one it likes best (usually the oldest) and then tells the projects to use it going forward.

```

Comment:
it is always the oldest. Using language like "which one [BOINC] likes best" is confusing

Reply:
Fixed